src/main/java/seedu/weeblingo/ui/ScoreHistoryListPanel.java

```diff
@@ -0,0 +1,44 @@
+package seedu.weeblingo.ui;
+
+import java.util.logging.Logger;
+
+import javafx.collections.ObservableList;
+import javafx.scene.control.ListCell;
+import javafx.scene.control.ListView;
+import javafx.scene.layout.Region;
+import seedu.weeblingo.commons.core.LogsCenter;
+import seedu.weeblingo.model.score.Score;
+
+public class ScoreHistoryListPanel extends UiPart<Region> implements ListPanel {
```

Comment:
Is there a reason for these 2 list panels to implement list panel? extending UiPart<Region> seems to be sufficient OOP for me :0

Reply:
Originally that Interface is to help with a generic method to change the content of `flashcardListPanelPlaceHolder`. Now the current version it is no longer needed. Will fix it thanks!